src/exporters/vhi.py

```diff
@@ -196,25 +167,38 @@ def check_failures(self) -> List:
 
         with open(pickled_error_fname, 'rb') as f:
             errors = pickle.load(f)
-        print("*************************")
-        print("Errors:")
-        pprint(errors)
 
         return errors
 
-    def export(self) -> List:
-        """Export VHI data from the ftp server
+    @staticmethod
+    def get_default_years():
+        """ returns the default arguments for no. years """
+        years = [yr for yr in range(1981, 2020)]
+
+        return years
+
+    def export(self, years: List = [yr for yr in range(1981, 2020)]) -> List:
```

Comment:
@gabrieltseng  does this still violate the problems here: https://github.com/esowc/ml_drought/pull/12#discussion_r285190313

If so does hte funciton above fix that?

Do we need to have 
`min_year` and `max_year` as arguments instead?